Makefile

```diff
@@ -22,7 +22,7 @@ endif
 include $(BOLOS_SDK)/Makefile.defines
 
 APP_LOAD_PARAMS  = --curve secp256k1
-ifeq ($(TARGET_NAME), TARGET_NANOX)
+ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_NANOX TARGET_STAX))
```

Comment:
From what I've understood, this is related to the BLE functonnality, should we use the `HAVE_BLE` instead ? 
Anyway, I think this should probably be moved to the SDK.

Reply:
Keeping these for your PR adding a lot of stuff in SDK